deploy.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2018 Ultimaker B.V.
+import os
+import logging
+import sys
+import time
+from dotenv import load_dotenv
+
+from CuraPackageDeployer.Config import Config
+from CuraPackageDeployer.CuraPackageDeployer import CuraPackageDeployer
+
+
+load_dotenv()
+
+
+class CuraDriveConfig(Config):
+    package_id = "CuraDrive"
+    package_sources_dir = "./CuraDrive"
+    tags = ["backups", "cloud", "restore", "configuration", "settings", "sync"]
+    website = "https://ultimaker.com"
+    release_notes = "Switched to the new Ultimaker Account functionality baked into Cura."
+    access_token = os.getenv("ACCESS_TOKEN", None)
```

Comment:
What happens if the access token is empty?

Reply:
The 1st API call will fail, so the CI will fail. The library is not 100% robust yet (that's why it's on my own account still) and definitely needs more validation, but then again it's an automation tool used by a small number of people.